tests/test_x_eclab.py

```diff
@@ -78,6 +89,59 @@ def test_eclab_consistency(froot, locale, datadir):
     compare_params(aret, bret)
 
 
+@pytest.mark.parametrize(
+    "froot, locale",
+    [
+        ("ca", "en_US"),
+        ("ca.issue_134", "en_US"),
```

Comment:
I think testing just one case is enough here, no need to waste minutes of CI redoing all the MPR tests.